pkg/p2p/peer/peer.go

```diff
@@ -28,9 +28,14 @@ type Parent struct {
 	SkipMessageList *messages.SkipMessageList
 }
 
-func NewParent() Parent {
+func NewParent(enableLightNode bool) Parent {
+	messageChSize := 100
+	if enableLightNode {
+		// because in light node we send block and snapshot request messages
+		messageChSize = 200
+	}
```

Comment:
I think `100` should be named constant. And if `enableLightNode == true` then `messageChSize` = `messageChSize * 2`